node/resolvers/sponsoredProducts.ts

```diff
@@ -69,8 +69,9 @@ export async function sponsoredProducts(
 
   try {
     const adResponse = await ctx.clients.adServer.getSponsoredProducts({
-      count: SPONSORED_PRODUCTS_COUNT,
       searchParams: getSearchParams(args, privateSellers),
+      count: args.sponsoredCount ?? DEFAULT_SPONSORED_COUNT,
+      placement: args.placement,
```

Comment:
🟡 
What do you think of also including here `channel`?

Reply:
I like the idea. I'll do it later, since now I want to deploy sponsored products on shelf and I would have to change 200 PR's to include the channel 🤡 .